### PR TITLE
MAINT: moving env_analysis plots and prints

### PR DESCRIPTION
**Pull request by Gui-FernandesBR**
## Pull request type

Please check the type of change your PR introduces:

- [x] Code maintenance (refactoring, formatting, renaming, tests)

## Pull request checklist

Please check if your PR fulfills the following requirements, depending on the type of PR:

- Code base maintenance (refactoring, formatting, renaming):

  - [x] Docs have been reviewed and added / updated if needed
  - [x] Lint (`black rocketpy`) has passed locally and any fixes were made
  - [x] All tests (`pytest --runslow`) have passed locally

## What is the current behavior?

The Environment Analysis class wasn't modified in the last plots/prints overhaul (see #312).

## What is the new behavior?

More than 1.5k code lines were moved from the EnvironmentAnalysis file to the corresponding plots and prints file.
This makes the library more readable, and also improve the consistency over the rocketpy classes.
The expected results should be the same, in the background the code is way more organized now.

## Does this introduce a breaking change?

- [x] Yes (The EnvironmentAnalysis class lost a lot of its methods, which now can be accessed by the `plots` and `prints` atributes)

## Other information
* Question: Do you want me to also update the 2 environment analysis notebooks? So far, I didn't updated it to
* Please do not skip tests locally... We need everything to be running flawlessly with the `--runslow` option included

### CODE REVIEW

**Comment by Gui-FernandesBR** [comment]
Quick comment: I was facing a little problem when running the class right before modifying everything. See:
```
AttributeError: module 'numpy' has no attribute 'float'.
`np.float` was a deprecated alias for the builtin `float`. To avoid this error in existing code, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
```

In order to solve it, I simply changed all the `np.float` appearances to `float`, just like recommended above. Everything should be running ok now.

Also, to run the new tests, I am using the following command on my vscode terminal:
```cmd
pytest tests/test_environment_anaysis.py --runslow
``` 

Notice how the test name is mistyped... It says "_anaysis.py" instead of "_analysis.py". 
A `git rename` command should solve it, but let's wait for the review first.

File: rocketpy/plots/environment_analysis_plots.py
Comment (Gui-FernandesBR) [inline rocketpy/plots/environment_analysis_plots.py:1]: All this file is already in snake_case. How great is that? 
😁😁

**Comment by MateusStano** [comment]
> Notice how the test name is mistyped... It says "_anaysis.py" instead of "_analysis.py". A `git rename` command should solve it, but let's wait for the review first.

I actually found some more problems with the env analysis tests. They don't have the `@patch("matplotlib.pyplot.show")` so all plots open a new window. I will open a quick new PR fixing the name and the tests

**Comment by MateusStano** [comment]
> * Question: Do you want me to also update the 2 environment analysis notebooks? So far, I didn't updated it to

Yes, please! Or else this PR feels incomplete

**Review by MateusStano — CHANGES_REQUESTED** [review]
Changes remaining:

- [ ] Update notebooks
- [ ] Fix `_anaysis` typo
- [ ] Add `@patch("matplotlib.pyplot.show")` decorator to allInfo test

**Comment by Gui-FernandesBR** [comment]
> > * Question: Do you want me to also update the 2 environment analysis notebooks? So far, I didn't updated it to
> 
> Yes, please! Or else this PR feels incomplete

Ok! I just didn't want to create a bunch of conflicts within notebooks. But let's go, you're the boss here :)

**Comment by Gui-FernandesBR** [comment]
@MateusStano , I updated everything as requested. Ready for a re-review now

**Review by MateusStano — APPROVED** [review]
All good! Feel free to merge whenever

**Pull request merged**